### ISSUE TITLE
SecurityCraft Crash with Build 14

### ISSUE THREAD

**Issue by TerdyTheTerd** [TCPR/Thermos#72]
Build 14 worked fine on  fresh install, however when testing it on an already pregenned and played on world I get [this](http://paste.ubuntu.com/14886487/). Really not sure what commit may have caused this to happen. This crash was still present when trying to update to build 13. 

P.S. Some of the crash report was omitted as it contained player cords, but it was just repeating the same info about the block trying to update. I left the first part of the first block in there(without the cords) all other instances were the same except with different cords.

**Comment by TerdyTheTerd** [comment]
I would like to close this, I didn't realize at first, but apparently, somehow, maybe the gods hate me, but simply switching server versions, not even changing any mods, remapped the level.dat id mappings, corrupting the world. Doesn't seem to be Thermos fault, just my own stupidity.

**Comment by Robotia** [comment]
Hmm, that is very unfortunate. I would be curious to see if it's possible to remap the world to match the new level data. Something similar has happened to me before.

**Comment by Cracks777** [comment]
Does this mean that old world will corrupt?

**Comment by Robotia** [comment]
@Cracks777 He means that he changed a mod and didn't realize that that's what caused it.

**Comment by Cracks777** [comment]
lol Oh ok

**Comment by TerdyTheTerd** [comment]
Really don't know why, I updated a few mods but didn't add or remove any. There should have been room for the few additional items that were added (no items were removed, forge never asked me to /fml confirm). Even when trying to start with the old mods and old KCauldron it still seems corrupt. These things happen sometimes, if only there was a way to prevent the fml id mappings from corrupting the entire world :/ I know when forge init it creates a map of id's, then when it loads the world it compares its list to the list stored in the level.dat and then injects those id's into forge.
